test/test.js

```diff
@@ -226,3 +226,45 @@ describe('setting up an initializing a local node', function () {
     })
   })
 })
+
+describe('change config values of a disposable node', function () {
+  this.timeout(20000)
```

Comment:
Basically, it's chosen because IPFS usually responds within those timeframes. In node-ipfs-api we have to do the same, since the response times varies... Kind of sucks